v2/pkg/subscraping/sources/zoomeyeapi/zoomeyeapi.go

```diff
@@ -47,14 +48,22 @@ func (s *Source) Run(ctx context.Context, domain string, session *subscraping.Se
 			return
 		}
 
+		randomApiInfo := strings.Split(randomApiKey, ":")
```

Comment:
I think the api key here has a different format:
![image](https://github.com/projectdiscovery/subfinder/assets/13421144/a4759774-e1c3-47aa-afcd-991f96c26d4c)

Reply:
Now, they've different hosts for different regions. Therefore, we ask users to add the host they're using in the provider config file. For example, https://github.com/projectdiscovery/docs/pull/38/files#diff-52512d3883ec5968f7eb443fe887c80b4563f990c466d14caa54e17d5b144b70R107